ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,13 @@
+<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨 -->
+
+<--
+Please fill in the *entire* template below.
+
+The template is designed to avoid unnecessary delays to confirm and fix reported issues. Issues with missing information generally ends up with a `need-info` label and take significally more time to fix. Please help us to help you!
```

Comment:
Issues ends up -> Issues end up

Also I think "help us help you" without the (to) is more common.